plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+
+
+class QueueType(Enum):
+    auto = "auto"
+    local = "local"
+    slurm = "slurm"
+
+
+@dataclass
+class SlurmQueueConf:
+    nodes: int = 1
+    gpus_per_node: int = 1
+    ntasks_per_node: int = 1
+    mem: str = "${hydra.launcher.mem_limit}GB"
+    cpus_per_task: int = 10
+    time: int = 60
+    partition: str = "dev"
+    signal_delay_s: int = 120
+    job_name: str = "${hydra.job.name}"
```

Comment:
try with `${hydra:job.name}`.
This is now the recommended form for Hydra config interpolation. It will also work inside the user's config.

Reply:
per offline discussion, keeping it as ```hydra.``` for now.